lib/coins/axe.js

```diff
@@ -0,0 +1,25 @@
+var common = {
+  name: 'Axecoin',
+  unit: 'AXE'
+}
+
+var main = Object.assign({}, {
+  hashGenesisBlock: '0x00000848c4c426413b0a42d4d127085f903670f51aca7b177d2c004867bc50af',
+  port: 29057,
+  portRpc: 29058,
+  protocol: {
+  },
+  versions: {
+    bip32: {
+    },
+    bip44: 7,
```

Comment:
7 belongs to namecoin
I do not see AXE here: https://github.com/satoshilabs/slips/blob/master/slip-0044.md